### DISCUSSION TITLE
[Feature] Show sequential images when split view enabled.

### DISCUSSION THREAD

**Discussion by Boyda-dev** [damongolding/immich-kiosk#111] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The new split view is great, but I keep seeing two very unrelated or mismatched photos side by side. Could the portrait images for split view have the option of being sequential portrait photos from the album or collection?

**Comment by damongolding** [comment]
This was a [consideration](https://github.com/damongolding/immich-kiosk/discussions/17#discussioncomment-10736805) but it was easier and faster to do random for the initial release. Very happy to revisit and refine :)

**Reply by Boyda-dev** [reply]
Very cool. Thank you for responding. Love the project.

**Comment by jschwalbe** [comment]
Once it showed two images of my same child side by side. A couple years apart. Blew my mind. :) then I wanted it to happen all the time. Haha
